### PR TITLE
Added Telegraf InfluxDB v2 Output Support

### PR DESCRIPTION
**Pull request by jojoxd**
Adds the feature from #1847

### CODE REVIEW

**Comment by jojoxd** [comment]
Do I need to add my name to the LICENSE file?

There doesn't seem to be a CLA at least.

File: net-mgmt/telegraf/pkg-descr [hunk 1]
```diff
@@ -11,6 +11,9 @@ Kafka, MQTT, NSQ, and many others.
 Plugin Changelog
 ================
 
+1.8.0
+* Add InfluxDB v2 output support
```
Comment (mimugmail) [inline net-mgmt/telegraf/pkg-descr:15]: Can you add a space here please

File: net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml [hunk 2]
```diff
@@ -35,6 +35,36 @@
...
+    </field>
+    <field>
+        <id>output.influx_v2_token</id>
+        <label>Influx v2 Token</label>
+        <type>text</type>
+        <help>Influx v2 authentication token.</help>
+    </field>
+    <field>
+        <id>output.influx_v2_organization</id>
+        <label>Influx v2 Organization Name</label>
```
Comment (mimugmail) [inline net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml:58]: You can remove `Name` and make help-text more descriptive

File: net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml [hunk 3]
```diff
@@ -35,6 +35,36 @@
...
+        <type>text</type>
+        <help>Influx v2 organization name.</help>
+    </field>
+    <field>
+        <id>output.influx_v2_bucket</id>
+        <label>Influx v2 Bucket Name</label>
```
Comment (mimugmail) [inline net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml:64]: You can remove `Name` and make help-text more descriptive

**Comment by mimugmail** [comment]
No need for license update when there are only changes in templating. Looks good beside some small cosmetic things, thanks :)

If you want to test you code you can just go to CLI of your firewall, `pkg install git` and `git clone http://github.com/jojoxd/plugins` then `cd plugins/net-mgmt/telegraf` and `git checkout feature/telegraf-influx2-output` then install via `make upgrade`

File: net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml (hunk 2 above)
Comment (jojoxd) [reply net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml]: should be a bit better now :)

File: net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml (hunk 3 above)
Comment (jojoxd) [reply net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml]: should be a bit better now :)

**Comment by jojoxd** [comment]
Just tested it, works great!

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**